integration_test/ops_agent_test.go

```diff
@@ -2300,6 +2303,51 @@ func TestPrometheusMetrics(t *testing.T) {
 		if multiErr != nil {
 			t.Error(multiErr)
 		}
+
+		expectedFeatures := []*feature_tracking_metadata.FeatureTracking{
+			{
+				Module:  "logging",
```

Comment:
Question: for prom we extract other features as well: https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/valid/linux/metrics-receiver_prometheus_relabel/golden/features.yaml
Do we normally include all features in the integration tests, or just check the representative ones is good?

Reply:
I figured we would just test what matter to us here since the rest are covered by the unit tests but alas I was wrong. The tests fail because even though all the metrics come up correctly (http://shortn/_9Yh3qHC1mT) but our tests require that all metrics are listed here so it knows how many times to call `gce.WaitForMetric` since all metrics write too the same metric type.

I fixed this now